modules/nordvpn.nix

```diff
@@ -0,0 +1,45 @@
+{ config, lib, pkgs, ... }:
+
+with lib; {
+  options.services.nordvpn.enable = mkOption {
```

Comment:
Instead of `services.nordvpn`, I think I prefer `chaotic.nordvpn` (so one easily knows what does not come from nixpkgs in its config), do you agree?